packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -23,6 +23,7 @@ interface ComboboxProps<T extends string | number>
   createCustomValue?: boolean;
   options: Array<ComboboxOption<T>>;
   onChange: (option: ComboboxOption<T> | null) => void;
+  onInputChange?: (inputValue: string) => void;
```

Comment:
I remember someone a while back saying we should avoid this ;) I guess this is something that should be the onChange for a `SuggestionInput` component that you demoed.

Reply:
Yeah, this'll all come down to whether we add a proper async `loadOptions` prop or not.

To be honest, the longer i play with this, the more i think that doing async select in userland might not be the best.